src/components/Form.js

```diff
@@ -62,33 +51,28 @@ const Form = () => {
           onChange={handleTitleChange}
           required
         />
-      </label>
-      <input
-        type="text"
-        id="author"
-        placeholder="Book author"
-        value={form.authorInput}
-        onChange={handleAuthorChange}
-        required
-      />
-      <select
-        value={form.selectedValue}
-        onChange={handleSelectChange}
-        name="categories"
-        id="categories"
-        required
-      >
-        <option value="none" disabled>
-          Category
-        </option>
-        <option value="education">Education and Teaching</option>
-        <option value="history">History</option>
-      </select>
-      <button className="btn" type="submit">
-        ADD BOOK
-      </button>
-    </form>
+        <select
+          value={form.selectedValue}
+          onChange={handleSelectChange}
+          name="categories"
+          id="categories"
+          required
+        >
+          <option value="none" disabled>
+            Category
+          </option>
+          <option value="education">Education</option>
+          <option value="history">History</option>
+          <option value="entertainment">Entertainment</option>
+          <option value="mystery">Fiction</option>
+          <option value="romance">Crime</option>
+          <option value="fantasy">True Story</option>
+        </select>
```

Comment:
- [ ] [OPTIONAL] This is a great implementation of the select form feature. There is also an option of storing the select options into an array where one can use the map method to loop through and provide select option values dynamically.